### ISSUE TITLE
md5 hash failing

### ISSUE THREAD

**Issue by tenacex** [xdissent/ievms#304]
I had to bypass the check md5 function to actually get your script to work. Not sure whether this is broken for everybody else? it basically just goes 

```
'MD5 check failed for Oracle_VM_VirtualBox_Extension_Pack-5.0.20.vbox-extpack (wanted , got ad35ea928dfbb2e0d46cd0c86b82e8e5)'

```

**Comment by grego3** [comment]
I also had this issue. I wound up just getting the extension pack from [VirtualBox.org](https://www.virtualbox.org/wiki/Downloads).

**Comment by junkystu** [comment]
PR #295 looks like it fixes this.

**Comment by gavD** [comment]
I'm still getting this even with #295

**Comment by gavD** [comment]
If I install the extension pack from https://www.virtualbox.org/wiki/Downloads first, it works fine (OSX El Capitan)

**Comment by minusfive** [comment]
Hmm... Still getting this, even after installing VB extension pack.

**Comment by axebox** [comment]
I got the failed MD5 and installed VB extension pack with an upgraded VirtualBox and still get the failed MD5.

ievms is supposed to be `no fuss`. Why would we use it if there's always errors of some form? Been like this for a year now. End rant.

**Comment by francescom** [comment]
Same problem here
MD5 check failed for MSEdge_Win10.zip (wanted c1011b491d49539975fb4c3eeff16dae, got 5347a5d19e4a1840d97a86531691ed18)
Failed to download https://az792536.vo.msecnd.net/vms/VMBuild_20150801/VirtualBox/MSEdge/Mac/Microsoft%20Edge.Win10.For.Mac.VirtualBox.zip to /Virtual Machines/ievms/MSEdge_Win10.zip (attempt 3 of 3)

@tenacex How do you bypass the check md5 function

**Comment by loucapo** [comment]
i have the exact same issue here. is this even being supported nowadays? is there any alternative?

**Comment by jayrosen1576** [comment]
It looks like (at least for MS Edge) the path tot he vm has changed. 

The _**correct**_ path is:
https://az792536.vo.msecnd.net/vms/VMBuild_20160322/VirtualBox/MSEdge/MSEdge.Win10TH2.VirtualBox.zip

The _**incorrect**_ current path for this repo is:
https://az792536.vo.msecnd.net/vms/VMBuild_20150801/VirtualBox/MSEdge/Mac/Microsoft%20Edge.Win10.For.Mac.VirtualBox.zip

You can simply download the vm using the correct url rather than rely on this automated script.

**Comment by interist32** [comment]
Same problem.

**Comment by interist32** [comment]
I've just updated my Virtual Box manually to 5.1 and everything works now.

Before that I tried to update Virtual Box using brew-cask and it gave no effect.
Hope it helps.

**Comment by Undistraction** [comment]
Same issue with VB 5.1: `MD5 check failed`

**Comment by rnicholus** [comment]
I'm running into this issue as well. It seems this project is dead/abandoned. The best course of action is to download the VMs directly from the modern.ie site.

**Comment by tmitry** [comment]
curl -s https://raw.githubusercontent.com/xdissent/ievms/master/ievms.sh | env IEVMS_VERSIONS="EDGE" bash

return:
MD5 check failed for MSEdge_Win10.zip (wanted c1011b491d49539975fb4c3eeff16dae, got a571bf44a7d38c1ab693f95f4d66cb26)
Failed to download https://az792536.vo.msecnd.net/vms/VMBuild_20150801/VirtualBox/MSEdge/Mac/Microsoft%20Edge.Win10.For.Mac.VirtualBox.zip to /Users/username/.ievms/MSEdge_Win10.zip (attempt 3 of 3)

**Comment by igorescobar** [comment]
Same issue here. 

```
⋊> ~ curl -s https://raw.githubusercontent.com/xdissent/ievms/master/ievms.sh | env IEVMS_VERSIONS="EDGE" bash                                                                       10:38:23
Checking for VirtualBox
Checking for Oracle VM VirtualBox Extension Pack
Pack no. 0:   Oracle VM VirtualBox Extension Pack
Building IE EDGE VM
Checking for existing OVA at /Users/igorescobar/.ievms/MSEdge - Win10.ova
Found OVA ZIP at MSEdge_Win10.zip - skipping download
MD5 check failed for MSEdge_Win10.zip (wanted c1011b491d49539975fb4c3eeff16dae, got a4f726c8e94a69bd167b1627c8b527da)
Check failed - redownloading OVA ZIP
Downloading OVA ZIP from https://az792536.vo.msecnd.net/vms/VMBuild_20150801/VirtualBox/MSEdge/Mac/Microsoft%20Edge.Win10.For.Mac.VirtualBox.zip to MSEdge_Win10.zip (attempt 1 of 3)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   215  100   215    0     0    204      0  0:00:01  0:00:01 --:--:--   204
MD5 check failed for MSEdge_Win10.zip (wanted c1011b491d49539975fb4c3eeff16dae, got 912572e4ef73aa65b7cec2f05d0b8e6c)
Redownloading OVA ZIP
Found OVA ZIP at MSEdge_Win10.zip - skipping download
MD5 check failed for MSEdge_Win10.zip (wanted c1011b491d49539975fb4c3eeff16dae, got 912572e4ef73aa65b7cec2f05d0b8e6c)
Check failed - redownloading OVA ZIP
Downloading OVA ZIP from https://az792536.vo.msecnd.net/vms/VMBuild_20150801/VirtualBox/MSEdge/Mac/Microsoft%20Edge.Win10.For.Mac.VirtualBox.zip to MSEdge_Win10.zip (attempt 2 of 3)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   215  100   215    0     0    670      0 --:--:-- --:--:-- --:--:--   669
MD5 check failed for MSEdge_Win10.zip (wanted c1011b491d49539975fb4c3eeff16dae, got 962652395e1da38ff05c93642c2d9a7d)
Redownloading OVA ZIP
Found OVA ZIP at MSEdge_Win10.zip - skipping download
MD5 check failed for MSEdge_Win10.zip (wanted c1011b491d49539975fb4c3eeff16dae, got 962652395e1da38ff05c93642c2d9a7d)
Check failed - redownloading OVA ZIP
Downloading OVA ZIP from https://az792536.vo.msecnd.net/vms/VMBuild_20150801/VirtualBox/MSEdge/Mac/Microsoft%20Edge.Win10.For.Mac.VirtualBox.zip to MSEdge_Win10.zip (attempt 3 of 3)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   215  100   215    0     0    656      0 --:--:-- --:--:-- --:--:--   657
MD5 check failed for MSEdge_Win10.zip (wanted c1011b491d49539975fb4c3eeff16dae, got 793da790e555a47d9da78955fe8f468a)
Failed to download https://az792536.vo.msecnd.net/vms/VMBuild_20150801/VirtualBox/MSEdge/Mac/Microsoft%20Edge.Win10.For.Mac.VirtualBox.zip to /Users/igorescobar/.ievms/MSEdge_Win10.zip (attempt 3 of 3)
```

**Comment by xdissent** [comment]
Yep, sorry about this, although the issue is not my fault and presents a moving target which is antithetical to the concept of the project in the first place. Also, I recently had to built a windows machine to support VR, which I can test my stuff on, so it's not as high on my priority list as it used to be. All the old shit still seems to work for me though. I'll pull my strings at MS and see if we can't get a similarly stable dl URL/checksum for win10. Sorry for not being more available, but honestly MS has done such a decent job on Edge that we just don't have the same concerns about all the quirks that once existed in the browsers IEVMS is supposed to help you with.

**Comment by igorescobar** [comment]
No need to be sorry @xdissent ! And thanks a lot for your time and effort! ❤️

**Comment by corysimmons** [comment]
@xdissent Appreciate your efforts and honesty, but I still kinda like tinkering around with old IE regardless of the current state of Edge. I hope Microsoft can fix this instead of just pointing everyone to their newest browser.

**Comment by jsugarman** [comment]
Am having similar issue but purely with the unarchiver download checksum

```
MD5 check failed for unar1.5.zip (wanted fbf544d1332c481d7d0f4e3433fbe53b, got ae4362c240eb7909fcda1ce2f26cfb36)
Check failed - redownloading unar
Downloading unar from http://theunarchiver.googlecode.com/files/unar1.5.zip to unar1.5.zip (attempt 3 of 3)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1578  100  1578    0     0  14562      0 --:--:-- --:--:-- --:--:-- 14611
MD5 check failed for unar1.5.zip (wanted fbf544d1332c481d7d0f4e3433fbe53b, got ae4362c240eb7909fcda1ce2f26cfb36)
Failed to download http://theunarchiver.googlecode.com/files/unar1.5.zip to /Users/jsugarman/.ievms/unar1.5.zip (attempt 3 of 3)
```
the site reports it unarchiver has moved to bitbucket

**Comment by lynndylanhurley** [comment]
@jsugarman - try installing from #308. I was able to get past this using the following (to install `10`, `11`, and `EDGE`):

```curl -s https://raw.githubusercontent.com/sergionsz/ievms/master/ievms.sh | env IEVMS_VERSIONS="10 11 EDGE" bash```

**Comment by jsugarman** [comment]
@lynndylanhurley, thanks for this. I ended up working round the problem by getting the unar binary from another source

**Comment by mamboer** [comment]
@jsugarman how?

**Comment by willthemoor** [comment]
@mamboer `$ brew install unar`

**Comment by xdissent** [comment]
All of these issues should be addressed in 0.3.2, which was just released. If not, please open a new more specific issue, since this one kinda covered a lot of ground. Thanks!!
